pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/NewLoadManager.java

```diff
@@ -0,0 +1,66 @@
+package com.yahoo.pulsar.broker.loadbalance;
+
+import com.yahoo.pulsar.broker.PulsarServerException;
+
+/**
+ * New proposal for a load manager interface which attempts to use more intuitive method names and provide a starting
+ * place for new load manager proposals.
+ */
+public interface NewLoadManager {
```

Comment:
Is there a more descriptive way to name this class? 😄  What would be the main differentiator from the "old" load manager?